# Waypoints for navigating to specific zone from center of map

1:
  robot1:
    assigned_item_type: "red"
    assigned_zone: "cyan"

  robot2:
    assigned_item_type: "green"
    assigned_zone: "purple"

  robot3:
    assigned_item_type: "blue"
    assigned_zone: "green"


2:
  robot1:
    assigned_item_type: "green"
    assigned_zone: "purple"

  robot2:
    assigned_item_type: "red"
    assigned_zone: "cyan"

  robot3:
    assigned_item_type: "blue"
    assigned_zone: "pink"

3:
  robot1:
    assigned_item_type: "blue"
    assigned_zone: "green"

  robot2:
    assigned_item_type: "red"
    assigned_zone: "cyan"

  robot3:
    assigned_item_type: "green"
    assigned_zone: "purple"
